index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<head>
+<title>Portfolio</title>
+ <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="stylesheet" href="css/index.css">
+<link rel="stylesheet" href="css/sliding-menu.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+<script src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
+</head>
+<body>
+
+  <div id="container">
```

Comment:
You have done a great so far, however, inculcating best Html practices like using semantic tags such as header tag could help you visualize better organized code for yourself and for others in the team